zeus.nix

```diff
@@ -0,0 +1 @@
+release.nix
```

Comment:
Just an FYI for future reference, this layer of indirection is not required.  Zeus lets you choose which `.nix` file to build.